lib/rom/processor/transproc.rb

```diff
@@ -99,6 +99,8 @@ def visit(attribute)
       def visit_attribute(attribute)
         if attribute.typed?
           t(:map_value, attribute.name, t(:"to_#{attribute.type}"))
+        elsif attribute.meta[:coercer]
+          t(:map_value, attribute.name, attribute.meta[:coercer])
         end
```

Comment:
what error you think we should put there? ROM::Mapper::Error.new('You cannot specify type and use block in the same time!')?